docs/blog/2022-03-10-zephyr-3-0-upgrade-prep.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Zephyr 3.0 Update Preparation"
+author: Pete Johanson
+author_title: Project Creator
+author_url: https://gitlab.com/petejohanson
+author_image_url: https://www.gravatar.com/avatar/2001ceff7e9dc753cf96fcb2e6f41110
+tags: [firmware, zephyr, core]
+---
+
+As preperation for completing the [work](https://github.com/zmkfirmware/zmk/pull/1143) to upgrade ZMK to [Zephyr 3.0](https://docs.zephyrproject.org/3.0.0/releases/release-notes-3.0.html), users with user config repositories who wish to avoid future build failures with their GitHub Actions workflows can take steps to adjust
+their repositories now.
+
+GitHub Actions needs to use our latest Docker image to ensure continued compatibility with the ZMK codebase on Zephyr 3.0 (and beyond). You should:
+
+- Open `.github/workflows/build.yml` in your editor/IDE
+- Change `zmkfirmware/zmk-build-arm:2.5` to `zmkfirmware/zmk-build-arm:latest` wherever it is found
```

Comment:
Is `latest` intentional? (The tag [does not currently exist](https://hub.docker.com/r/zmkfirmware/zmk-build-arm/tags?page=1&name=latest)). 

Should we instead be advising users to switch to [`stable`](https://hub.docker.com/r/zmkfirmware/zmk-build-arm/tags?page=1&name=stable)?

Reply:
See, this is why review is so important. Brainfarts. Plain and simple.

Reply:
Fixed.